pages/chile.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="../style.css">
+    <link rel="shortcut icon" href="../assets/chile.jpg">
+    <title>CHILE</title>
+</head>
+
+<body style="display: inline-flexbox;">
+    <a href="../index.html">Back</a>
+    <h1>Chile Flag Tutorial</h1>
+    <div class='flag-flex'>
```

Comment:
You can use "horizontalStyle" at div on line 17, instead use flex-direction class.